app/controllers/posts_controller.rb

```diff
@@ -31,4 +31,12 @@ def create
 
     redirect_to user_posts_path
   end
+
```

Comment:
- [ ] it would be better if you add ` load_and_authorize_resource` to the postsController in order to authorize the resource which is the post as required

Reply:
Adressed